animations/lib/main.dart

```diff
@@ -21,6 +21,7 @@ import 'src/misc/expand_card.dart';
 import 'src/misc/focus_image.dart';
 import 'src/misc/physics_card_drag.dart';
 import 'src/misc/repeating_animation.dart';
+import 'src/misc/curved_animation.dart';
```

Comment:
@aksayushx Can you order the import alphabetically? This is why your Travis build is failing.

Reply:
Cool. I will do that